src/reducers/url-state.js

```diff
@@ -481,6 +481,25 @@ const localTrackOrderByPid: Reducer<Map<Pid, TrackIndex[]>> = (
   }
 };
 
+const localTrackOrderChangedByPid: Reducer<Set<Pid>> = (
+  state = new Set(),
+  action
+) => {
+  switch (action.type) {
+    case 'CHANGE_LOCAL_TRACK_ORDER': {
+      const localTrackOrderChangedByPid = new Set(state);
+      localTrackOrderChangedByPid.add(action.pid);
+      return localTrackOrderChangedByPid;
+    }
+    case 'SANITIZED_PROFILE_PUBLISHED':
+      // In localTrackOrderChangedByPid above the state is reset in this case,
+      // let's reset it here as well.
+      return action.oldThreadIndexToNew ? new Set() : state;
```

Comment:
I'm not 100% sure about this, but I believe this isn't completly wrong at least until I finally finish #3977...